wagtail/users/views/bulk_actions/user_bulk_action.py

```diff
@@ -17,9 +19,15 @@ def get_all_objects_in_listing_query(self, parent_id):
         listing_objects = self.model.objects.all().values_list("pk", flat=True)
         if "q" in self.request.GET:
             q = self.request.GET.get("q")
-            model_fields = {f.name for f in self.model._meta.get_fields()}
-            conditions = get_users_filter_query(q, model_fields)
+            if class_is_indexed(self.model):
+                search_backend = get_search_backend()
+                listing_objects = search_backend.search(
+                    q, User.objects.filter(self.group_filter)
```

Comment:
Won't this suffer from the same issue as the listing view (we can't filter search results by group)?